shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/ManagedBridgeServiceImpl.java

```diff
@@ -1,107 +1,20 @@
 package com.redhat.service.smartevents.shard.operator.v2;
 
-import java.util.Base64;
+import com.redhat.service.smartevents.shard.operator.v2.resources.ManagedBridge;
+import io.fabric8.kubernetes.client.KubernetesClient;
 
 import javax.enterprise.context.ApplicationScoped;
 import javax.inject.Inject;
-
-import com.redhat.service.smartevents.infra.v2.api.models.dto.BridgeDTO;
-import com.redhat.service.smartevents.shard.operator.core.providers.GlobalConfigurationsConstants;
-import com.redhat.service.smartevents.shard.operator.core.providers.TemplateImportConfig;
-import com.redhat.service.smartevents.shard.operator.core.providers.TemplateProvider;
-import com.redhat.service.smartevents.shard.operator.core.utils.LabelsBuilder;
-import com.redhat.service.smartevents.shard.operator.v2.providers.NamespaceProvider;
-import com.redhat.service.smartevents.shard.operator.v2.resources.KafkaConfigurationSpec;
-import com.redhat.service.smartevents.shard.operator.v2.resources.ManagedBridge;
-import com.redhat.service.smartevents.shard.operator.v2.resources.TLSSpec;
-
-import io.fabric8.kubernetes.api.model.Secret;
-import io.fabric8.kubernetes.client.KubernetesClient;
+import java.util.List;
 
 @ApplicationScoped
 public class ManagedBridgeServiceImpl implements ManagedBridgeService {
```

Comment:
@vaibhavjainwiz I'd like to leave this logic intact until we get things working with the v2 DTOs. At that point we can start to make more aggressive refactoring if we want to, but I want to limit the number of changes we're pushing in to start with.

Reply:
done.. I had made the requested changes.